### PR TITLE
Added docs for `unreceived_acknowledgements` and `unreceived_packets`

### PR DESCRIPTION
**Pull request by adizere**
This answers a user's question from Discord.
I originally wanted to document these details in the guide, but opted for code-level docs since this seems more appropriate.

Tagged @seanchen1991 as reviewer since we're both monitoring Discord and he has the most context.

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

**Review by mzabaluev — APPROVED** [review]
Looks good!

File: relayer/src/chain/counterparty.rs [hunk 1]
```diff
@@ -399,6 +399,29 @@ pub fn unreceived_acknowledgements_sequences(
         .map_err(Error::relayer)
 }
 
+/// This method expects an [`IdentifiedChannelEnd`] plus a pair of
```
Comment (seanchen1991) [inline relayer/src/chain/counterparty.rs:402]: I would flip the order of these first two sentences; start off with what this method does, and then mention what the method expects.

File: relayer/src/chain/counterparty.rs [hunk 2]
```diff
@@ -451,6 +474,34 @@ pub fn acknowledgements_on_chain(
     Ok((sequences, height))
 }
 
+/// This method expects an [`IdentifiedChannelEnd`] plus a pair of
```
Comment (seanchen1991) [inline relayer/src/chain/counterparty.rs:477]: Same here. I think what the method does should be specified first, and then mention the types that the method expects.

**Review by seanchen1991 — CHANGES_REQUESTED** [review]

**Comment by adizere** [comment]
Thanks @mzabaluev and @seanchen1991 !

@seanchen1991 Please re-review and merge if you agree.

**Review by seanchen1991 — APPROVED** [review]

**Pull request merged**